src/materials/Material.js

```diff
@@ -78,6 +78,8 @@ Material.prototype = Object.assign( Object.create( EventDispatcher.prototype ),
 
 	isMaterial: true,
 
+	onNodeBuild: function ( /* shaderobject, renderer */ ) {},
```

Comment:
I think I would go for `onBuild()`. Maybe other developers can use it for their own material system (which are probably not node-based).

Reply:
Ok, It was a sugestion only... I will change 👍